docs/getting-started/creating-your-first-automated-test.mdx

```diff
@@ -0,0 +1,132 @@
+---
+sidebar_label: 'Creating Your First Automated Test'
+---
+
+# Creating Your First Automated Test
```

Comment:
I think this is not new text, so will hold of on giving feedback on this

Reply:
Yeah this was renamed, moved from its previous url, and I updated some wording here and there but the content itself is old